install-interflop.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+REPO=interflop
```

Comment:
I'm not sure we should necessarily depend on the external interflop repository. Will it be maintained in the long run ?

If I understand correctly, for the moment this is only to retrieve the common header files ? The backends seem to be still be handled within verificarlo.

Moreover, on the technical side, why use here direct git commands instead of submodules (such as what we did with significantdigits) ?

Reply:
Yes, you're right. I included the interflop-stdlib in Verificarlo.